tests/test_celebrity.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.internal.celebrity import get_today_month_and_day
+from hypothesis import given
+from hypothesis import strategies as st
+
+
+CELEBRITY_PAGE = "/celebrity"
+DATE_START = datetime.date(2019, 1, 1)
+DATE_END = datetime.date(2022, 1, 1)
+
+
+@given(fake_date=st.dates(DATE_START, DATE_END))
+def test_get_today_month_and_day(fake_date):
+    if datetime.date.today().strftime("%m-%d") == fake_date.strftime("%m-%d"):
```

Comment:
`datetime.date.today().strftime("%m-%d") == get_today_month_and_day()`?

Isn't that what the function actually returns? ^^"

Reply:
Haha help please, I don’t understand how to test it then